app/assets/stylesheets/layout_components/forms.scss

```diff
@@ -215,6 +215,17 @@ input[type="checkbox"]{
   }
 }
 
+.date-pair{
```

Comment:
This is so much better! A while back I remember thinking about how odd this looked before.